frontends/ast/ast.h

```diff
@@ -355,6 +358,9 @@ namespace AST
 	std::pair<std::string,std::string> split_modport_from_type(std::string name_type);
 	AstNode * find_modport(AstNode *intf, std::string name);
 	void explode_interface_port(AstNode *module_ast, RTLIL::Module * intfmodule, std::string intfname, AstNode *modport);
+
+	// Helper for setting the src attribute.
+	void set_src_attr(RTLIL::AttrObject *obj, AstNode *ast);
```

Comment:
Minor nit: Can we mark `ast` `const` to clear any potential ambiguity?

Reply:
OK, done